### DISCUSSION TITLE
how to use mutiple st7789

### DISCUSSION THREAD

**Discussion by pasagame** [Bodmer/TFT_eSPI#2853] [Q&A - General] [answered]
how to use mutiple st7789, i watch clip on youtube but no detail. thank you.
https://youtu.be/Ug4_Wo1RWBU?si=LKDr11Id-w2SaSxc

**Comment by Bodmer** [comment, marked as answer]
Try this tutorial:
https://www.xtronical.com/multiplescreens/
